pkg/operator/ceph/file/subvolumegroup/controller.go

```diff
@@ -296,17 +297,21 @@ func (r *ReconcileCephFilesystemSubVolumeGroup) updateClusterConfig(cephFilesyst
 		},
 	}
 
+	csiHostNetworkEnabled, err := strconv.ParseBool(k8sutil.GetValue(r.opConfig.Parameters, "CSI_ENABLE_HOST_NETWORK", "false"))
+	if err != nil {
+		return errors.Wrap(err, "failed to parse value for 'CSI_ENABLE_HOST_NETWORK'")
+	}
 	// If the cluster has Multus enabled we need to append the network namespace of the driver's
 	// holder DaemonSet in the csi configmap
-	if cephCluster.Spec.Network.IsMultus() {
-		netNamespaceFilePath, err := csi.GenerateNetNamespaceFilePath(r.opManagerContext, r.client, cephCluster.Name, r.opConfig.OperatorNamespace, csi.CephFSDriverShortName)
+	if cephCluster.Spec.Network.IsMultus() || !csiHostNetworkEnabled {
+		netNamespaceFilePath, err := csi.GenerateNetNamespaceFilePath(r.opManagerContext, r.client, cephCluster.Namespace, r.opConfig.OperatorNamespace, csi.CephFSDriverShortName)
```

Comment:
You need to pass cephclusterName not its namespace please check this one.

Reply:
https://github.com/rook/rook/blob/ab31635d9b98e70eca042cda124cac5747d573da/pkg/operator/ceph/csi/spec.go#L911-L917

`GenerateNetNamespaceFilePath` func parameter tells to pass the clusterNamespace.
And also the holder template takes clusterNamespace.

https://github.com/rook/rook/blob/ab31635d9b98e70eca042cda124cac5747d573da/pkg/operator/ceph/csi/template/cephfs/csi-cephfsplugin-holder.yaml#L46-L52

Reply:
Yeah, you are correct, then we have a bug can you please address that in a different PR or commit?

Reply:
Agreed, let's open a separate PR for this fix

Reply:
Deleting this commit and opening new PR - https://github.com/rook/rook/pull/13663